### PR TITLE
Fix download url generation in alpharatio, danishbits, gft, speed.cd

### PR DESCRIPTION
**Pull request by miigotu**
Fixes https://github.com/SickRage/sickrage-issues/issues/818

/slap @labrys lol

Ofc this could use urljoin, but for now I just need them to actually work.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by labrys** [comment]
@miigotu While I understand that this is a fix to a bug, so ofc it should be merged so that the providers are working for now, the real fix should be to fix the later url generation in the code so that its generated properly.  A base url should not include the base path '/' appended to the end.  It should be part of the path as returned by urlparse:

From the Python docs:

``` python
>>> from urlparse import urlparse
>>> o = urlparse('http://www.cwi.nl:80/%7Eguido/Python.html')
>>> o   
ParseResult(scheme='http', netloc='www.cwi.nl:80', path='/%7Eguido/Python.html',
            params='', query='', fragment='')
```

**Comment by miigotu** [comment]
This is a temp revert until it can be done right with urljoin in the parser. Anything that works is better than something that doesnt work ^^^

File: sickbeard/providers/torrentbytes.py [hunk 1]
```diff
@@ -51,10 +51,10 @@ def __init__(self):
         self.url = 'https://www.torrentbytes.net'
         self.urls = {
             'base_url': self.url,
-            'login': 'https://www.torrentbytes.net/takelogin.php',
-            'detail': 'https://www.torrentbytes.net/details.php?id=%s',
-            'search': 'https://www.torrentbytes.net/browse.php?search=%s%s',
-            'download': 'https://www.torrentbytes.net/download.php?id=%s&name=%s'
+            'login': self.url + 'https://www.torrentbytes.net/takelogin.php',
```
Comment (Waty) [inline sickbeard/providers/torrentbytes.py:54]: Typo?

**Pull request merged**